sdk/iot/provisioning/samples/src/paho_iot_provisioning_example.c

```diff
@@ -13,13 +13,17 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#ifdef _WIN32
```

Comment:
When is this true and is `unistd.h` not available on windows? Maybe add a comment?

Reply:
What do you mean? Win32 is Windows, Uni is Posix. This is very common, so I'd rather not add a comment.

Reply:
I am not familiar with this header. My question is: Why do we need to do this? What are we doing that is OS/platform specific?

To clarify: I wasn't asking about what Win32 means or suggesting we comment that :p

Reply:
Oh, I get it now. I'll add that comment. This is necessary for the sleep function.